client/blocks/login/login-form.jsx

```diff
@@ -428,6 +430,33 @@ export class LoginForm extends Component {
 		);
 	}
 
+	renderAlreadyLoggedInPrompt() {
+		const { isLoggedIn, currentUserName } = this.props;
+		const { isFormDisabledWhileLoading } = this.state;
+
+		return (
+			! isFormDisabledWhileLoading &&
+			isLoggedIn && (
+				<Notice status="is-info" showDismiss={ false } icon="lock">
+					{ this.props.translate(
+						'You are already logged in as {{em}}%(email)s{{/em}}. ' +
+							'{{a}}Click here{{/a}} to continue as that user, or log in as ' +
+							'a different user below.',
+						{
+							args: {
+								email: currentUserName,
+							},
+							components: {
+								a: <a href="/" />,
+								em: <em />,
+							},
+						}
+					) }
+				</Notice>
+			)
+		);
+	}
```

Comment:
The notice is not particularly beautiful and the text is very long. In #34541, with @davemart-in we added a subtle alternative action "or visit [some site]" under login header. It could be adapted like this:

<img width="421" alt="Screenshot 2019-08-02 at 16 19 02" src="https://user-images.githubusercontent.com/664258/62376680-80ce0d80-b541-11e9-8a82-0c174b0e76b8.png">

Adding a little gravatar next to my name would be a nice finishing touch. What do you think?

Reply:
See, this is why I'm an engineer and not a designer 😄 I'll work on those changes, thanks!

Reply:
Done!

![image](https://user-images.githubusercontent.com/409615/62787723-3a3a5f00-babd-11e9-9e63-ad885f49cd5d.png)